src/com/flickity

```diff
@@ -0,0 +1,88 @@
+/**
+ * @author Deux Huit Huit
+ *
+ *	Flickity component
+ */
+
+(function ($, win, undefined) {
+
+	'use strict';
+
+	App.components.exports('flickity', function (options) {
+		var slider = $();
+		var scope = $();
+
+		var defaultOptions = {
+			sliderCtn: '.js-flickity-slider-ctn',
+			cellSelector: '.js-flickity-cell',
+			navBtnSelector: '.js-flickity-nav-btn',
+
+			abortedClass: 'is-flickity-cancelled',
+			initedClass: 'is-flickity-inited',
+			selectedClass: 'is-selected',
+
+			dataAttrPrefix: 'data-flickity-'
+		};
+
+		var o = $.extend({}, defaultOptions, options);
+
+		var flickityOptions = function () {
+			var opts = {};
+
+			var dataAttrPattern = new RegExp('^' + o.dataAttrPrefix);
+
+			var attrs = _.omit(slider[0].attributes, function (key, value) {
+				return !dataAttrPattern.test(key.name);
+			});
+
+			_.each(attrs, function (key) {
+
+				var val = key.value;
+
+				if (!_.isNaN(parseFloat(val, 10))) {
```

Comment:
Unlike `parseInt()` floats can only be base 10. Also, please used a temp variable and store the result of the `parseFloat()` call.